docs/_asset/index.css

```diff
@@ -789,11 +791,54 @@ button.success {
   );
 }
 
+.react-tabs__tab-panel {
+}
+
+.react-tabs__tab-panel--selected {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+/* Preview / Options */
+.react-tabs__tab-panel--selected.tab-panel-scrollable {
+  overflow: auto;
+}
+
 .frame-body {
   border-bottom-left-radius: 3px;
   border-bottom-right-radius: 3px;
 }
 
+.react-tabs__tab-panel--selected > .frame-body {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
This selector applies to the options, making them rather big

Reply:
Oh yes, this is because margin collapsing only applies to block and not flex

Reply:
Fixed, also made them slightly smaller, will probably be better for introducing new options

<img width="458" alt="CleanShot 2023-05-10 at 19 14 44@2x" src="https://github.com/mdx-js/mdx/assets/749374/f8305fa9-5896-4359-8862-f21c8cc2634b">